### PR TITLE
API provides functionality to create watchlist

### PR DESCRIPTION
**Pull request by Alex-Bellamy**
### User Story
```
As a movie API
In order for subscriber to watch movies later
I want to provide the functionality to create a watchlist
```
## Changes proposed in this pull request:
* Ability for a subscriber to create a watchlist
## What I have learned working on this feature:
* Using conditional statements to allow only subscribers to add to a watchlist
## Packages/Gems added
- Devise_token_auth

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```
Comment (Cma-B) [inline app/models/user.rb:1]: I guess. you need to remove outcommented line and extra lines 1 and 2

**Review by Cma-B — APPROVED** [review]
Good job!

File: app/controllers/api/v1/watchlist_items_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Api::V1::WatchlistItemsController < ApplicationController
+  before_action :authenticate_user!
+  def create
+    if current_user.subscriber
+      if current_user.watchlist.nil?
+         watchlist = Watchlist.create(user_id: current_user.id)
```
Comment (viktorsommar) [inline app/controllers/api/v1/watchlist_items_controller.rb:6]: The indentation seems slightly off here

**Review by viktorsommar — COMMENTED** [review]
Just a comment about the indentation, then you are good for me

**Pull request merged**